src/components/flow/actions/helpers.tsx

```diff
@@ -70,6 +71,20 @@ export const renderIssue = (
     );
   }
 
+  if (issue.type === FlowIssueType.INVALID_LINK) {
+    message = (
+      <>
+        {i18n.t('issue.link_beginning', 'Link ')}
+        {issue.actual_link}
+        {i18n.t(
+          'issue.link_main',
+          ' can not be shortened because it is mistyped. Expected link is '
```

Comment:
@jacobmodel Can you please review the language here? Thank you.

Reply:
_can not be shortened because it is an invalid link. The expected link is_

@teehamaral